<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="0" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7" started-at="2019-01-14T15:37:22Z" finished-at="2019-01-14T15:37:22Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7" started-at="2019-01-14T15:37:22Z" finished-at="2019-01-14T15:37:22Z">
      <class name="Maven_kuber.ScriptsDriver">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:Maven_kuber.ScriptsDriver@553f17c]" name="beforeSuite" is-config="true" duration-ms="48" started-at="2019-01-14T15:37:22Z" finished-at="2019-01-14T15:37:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="FAIL" signature="endSuit()[pri:0, instance:Maven_kuber.ScriptsDriver@553f17c]" name="endSuit" is-config="true" duration-ms="0" started-at="2019-01-14T15:37:22Z" finished-at="2019-01-14T15:37:22Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at Maven_kuber.ScriptsDriver.endSuit(ScriptsDriver.java:91)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:455)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endSuit -->
      </class> <!-- Maven_kuber.ScriptsDriver -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
